hail/src/main/scala/is/hail/io/gen/LoadGen.scala

```diff
@@ -113,7 +114,7 @@ case class MatrixGENReader(
   skipInvalidLoci: Boolean) extends MatrixHybridReader {
 
   files.foreach { input =>
-    if (!HailContext.get.hadoopConf.stripCodec(input).endsWith(".gen"))
+    if (!HailContext.get.sFS.stripCodec(input).endsWith(".gen"))
```

Comment:
This HailContext.get.* vs HailContext.* is just inconsistent use, kept as is for simplicity in PR